run.sh

```diff
@@ -50,9 +50,6 @@ if [ "$INITIALIZE_TOOL_DEPENDENCIES" -eq 1 ]; then
     python ./scripts/manage_tool_dependencies.py init_if_needed
 fi
 
-[ -n "$GALAXY_UWSGI" ] && APP_WEBSERVER='uwsgi'
-find_server "${GALAXY_CONFIG_FILE:-none}" galaxy
```

Comment:
I think only the first of these 2 lines should be deleted, the other is needed to set the `run_server` variable.